specdb/build/privatedb.py

```diff
@@ -425,30 +436,35 @@ def ingest_spectra(hdf, sname, meta, max_npix=10000, chk_meta_only=False,
     return
 
 
-def mk_db(trees, names, outfil, ztbl, **kwargs):
+def mk_db(dbname, tree, outfil, ztbl, version='v00', **kwargs):
     """ Generate the DB
 
     Parameters
     ----------
-    trees : list
-      List of top level paths for the FITS files
-    names : list
-      List of names for the various datasets
+    dbname : str
+      Name for the database
+    trees : str
+      Path to top level of the tree of FITS files
```

Comment:
Maybe add to the docstring that directories with .fits files should not have subdirectories within. Also could add that the branches should be single instrument setups (right?)